components/UseCaseCard.vue

```diff
@@ -1,19 +1,32 @@
 <template>
-  <div class="flex p-4 shadow-xl">
+  <div class="flex flex-row overflow-auto p-4 shadow-xl">
     <!-- Usecase cards -->
-    <div
-      v-for="card in panel"
-      :key="card.id"
-      class="flex no-wrap text-left"
-    >
-      <NuxtLink :to="card.route">
-        <h4>
-          {{ card.title }}
-        </h4>
-        <div class="m-2 bg-white rounded">
-          <img :src="card.image" class="object-contain w-40 h-30 mx-auto">
-        </div>
-      </NuxtLink>
+    <div class="no-wrap text-left">
+      <h4 class="text-2xl m-2">
+        {{ title }}
+      </h4>
+      <p class="text-left m-2">
+        {{ description }}
+      </p>
+    </div>
```

Comment:
Can we make these text blocks link to the usecase main page as well?

Reply:
Done!